src/test/regress/sql/aggregate_support.sql

```diff
@@ -179,6 +179,12 @@ select floor(val/2), corr(valf, valf + val) from aggdata group by floor(val/2) o
 select floor(val/2), corr(valf, valf + val) from aggdata group by floor(val/2) having corr(valf + val, val) < 1 order by 1;
 select array_agg(val order by valf) from aggdata;
 
+-- Test alongside window functions
```

Comment:
Don't think these 4 test cases add anything existing tests don't already cover. Should be removed